index.js

```diff
@@ -26,6 +26,7 @@ const searchUrl = `${mapzenUrl}/search`
  *   {@link https://mapzen.com/documentation/search/autocomplete/#layers|layer types}
  * @param  {string} [$0.sources='gn,oa,osm,wof']
  * @param  {string} $0.text                       query text
+ * @param {string} $0.url                         optional URL to override Mapzen autocomplete endpoint
```

Comment:
Please format the jsdoc with the optional parameter style.  The `sources` parameter is formatted in that way.